spec.emu

```diff
@@ -350,7 +350,7 @@ contributors: Ujjwal Sharma, Younies Mahmoud
         1. Let _matcher_ be ? GetOption(_options_, *"localeMatcher"*, *"string"*, &laquo; *"lookup"*, *"best fit"* &raquo;, *"best fit"*).
         1. Let _numberingSystem_ be ? GetOption(_options_, *"numberingSystem"*, *"string"*, *undefined*, *undefined*).
         1. If _numberingSystem_ is not *undefined, then
-          1. If _numberingSystem_ does not match the Unicode Locale Identifier `type` nonterminal, throw a `RangeError` exception.
+          1. If _numberingSystem_ does not match the Unicode Locale Identifier `type` nonterminal, throw a *RangeError* exception.
```

Comment:
Regarding this and the previous two commits, I believe I switched from asterisks to backticks on @rbuckton's advice. That said, I'd be happy to do either as long as we have a clear contender.

Reply:
The style changed at some point &rarr; <https://github.com/tc39/ecma402/pull/396>, which links to other discussions and the corresponding PR for ECMA-262.